### DISCUSSION TITLE
initdb from a pg_dump file

### DISCUSSION THREAD

**Discussion by supermario18b** [cloudnative-pg/cloudnative-pg#3650] [General]
Hi everyone,
I'd like to restore a dump took with pg_dump when I initialize a new cloudnative-pg cluster.
I thought it is possible in the `bootstrap` section to define a dump file (ex. `somedbdump.sql`) and get the database in the cluster initialized with that.
Thanks in advance for any clarification,

supermario18b

**Comment by gbartolini** [comment]
Unfortunately, this feature is not currently available. Also, it might not be easy to implement, unless we use a dump generated by CloudNativePG somehow (see #3318). Once this is available, you could recover using pg_restore using an initdb bootstrapped cluster (or we could implement to counterpart of the dump command for the plugin).

If you look for something declarative, we need a way first to mount a volume in the init job that contains a dump file. Feasible, but so far we have had other priorities.

**Reply by supermario18b** [reply]
@gbartolini  Thank you for the clarification.
